openverse_catalog/dags/providers/provider_api_scripts/smithsonian.py

```diff
@@ -186,6 +190,48 @@ def _gather_unit_sample(unit, sample_dir, retries=RETRIES, endpoint=SEARCH_ENDPO
             break
 
 
+def get_new_and_outdated_unit_codes(unit_code_set, sub_prov_dict=SUB_PROVIDERS):
```

Comment:
Nit: A number of these variables are suffixed with the type (e.g. `_set`, `_dict`). It might be better to remove the suffixes and add type annotations to the function instead.